test/api/from-url-in-browser.js

```diff
@@ -0,0 +1,58 @@
+/* globals location:false */
+"use strict";
+const { assert } = require("chai");
+const { describe, it } = require("mocha-sugar-free");
+
+const { JSDOM } = require("../..");
+
+require("chai").use(require("../chai-helpers.js"));
+
+describe("API: JSDOM.fromURL()", { skipUnlessBrowser: true, timeout: 5000 }, () => {
```

Comment:
Instead of duplicating tests from from-url.js, please enable them in from-url.js instead. (By removing the top-level `{ skipIfBrowser: true }` and then adding it back for tests that don't pass in the browser.)

Reply:
Yes, I merged them.